CesiumGeometry/src/Axes.cpp

```diff
@@ -0,0 +1,65 @@
+#include "CesiumGeometry/Axes.h"
+#include <glm/mat3x3.hpp>
+
+namespace CesiumGeometry {
+
+namespace {
```

Comment:
I think unnamed namespace should be enough to make the function visible only in a local file. Static keyword maybe redundant

Reply:
That's right, according to my (albeit a bit vague) understanding of https://stackoverflow.com/q/13396748 (that "internal vs. external linkage" should not be relevant here...)